### PR TITLE
PLT-1848 Stats for recently active and new users now shows date with the time

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: web/react/components/admin_console/analytics.jsx [hunk 1]
```diff
@@ -378,7 +378,7 @@ export default class Analytics extends React.Component {
                                                     </time>
                                                 </OverlayTrigger>
                                             </td>
-                                            <td>{Utils.displayDateTime(user.create_at)}</td>
+                                            <td>{Utils.displayCommentDateTime(user.create_at)}</td>
```
Comment (enahum) [inline web/react/components/admin_console/analytics.jsx]: This should be with loc?

Comment (jwilander) [inline web/react/components/admin_console/analytics.jsx]: Good point, updating..

**Comment by hmhealey** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**